packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -18,7 +18,7 @@ class UpdatePackagesCommand extends FlutterCommand {
   UpdatePackagesCommand({ this.hidden: false }) {
     argParser.addFlag(
       'upgrade',
-      help: 'Run "pub upgrade" rather than "pub get".',
+      help: 'Run "flutter packages upgrade" rather than "flutter packages get".',
```

Comment:
This is describing what the command does.  It runs pub upgrade or pub get under the covers, it just happens to also set FLUTTER_ROOR (which is all `flutter packages *` does anyway).